modules/react/button/lib/DeleteButton.tsx

```diff
@@ -1,43 +1,44 @@
 import * as React from 'react';
-import {ButtonColors} from './types';
-import {BaseButton, BaseButtonProps, getMinWidthStyles, getPaddingStyles} from './BaseButton';
-import {
-  GrowthBehavior,
-  useTheme,
-  Themeable,
-  EmotionCanvasTheme,
-  createComponent,
-} from '@workday/canvas-kit-react/common';
 
-export interface DeleteButtonProps extends Themeable, GrowthBehavior, BaseButtonProps {
-  /**
-   * The size of the Button.
-   */
-  size?: 'small' | 'medium' | 'large';
-  children?: React.ReactNode;
-}
+import {buttonVars} from './BaseButton';
+import {createComponent} from '@workday/canvas-kit-react/common';
+import {createStyles, cssVar} from '@workday/canvas-kit-styling';
+import {base, brand, system} from '@workday/canvas-tokens-web';
+import {Button, ButtonProps} from './Button';
 
-const getDeleteButtonColors = ({
-  canvas: {
-    palette: {error: themeError},
-  },
-}: EmotionCanvasTheme): ButtonColors => ({
-  default: {
-    background: themeError.main,
-    label: themeError.contrast,
-  },
-  hover: {
-    background: themeError.dark,
+/**
+ * Extends all the style properties from Box to our buttons as well as props from ButtonContainerProps.
+ * We omit `ref` since all of our buttons use `createComponent` and already give access to `ref`.
+ * Use this type to extend and customize any one off buttons that you want full control over styling.
+ */
```

Comment:
I don't think you need this comment here, since the original prop interface has it. Same for the other buttons

Reply:
If this is removed, then DeleteButtonProps won't have JSDocs for it. You'd have to hover over ButtonProps directly. Is that what we want to do ?